### DISCUSSION TITLE
There is no xinput version of UE4ss?

### DISCUSSION THREAD

**Discussion by JRDK92** [UE4SS-RE/RE-UE4SS#381] [General]
https://www.youtube.com/watch?v=znqukVJKpqQ
This tutorial says to use Xinput version, can't find it though. Is it gone now?

**Comment by UE4SS** [comment]
That's an older version of UE4SS.
It's still available if you want it for some reason but I'd recommend using 3.0 instead which uses dwmapi.dll & UE4SS.dll instead of xinput1_3.dll.
